### ISSUE TITLE
As a player, I want an estimate of my ranking for the highest scoring unique QR code. (US 07.02.01)

### ISSUE THREAD

**Issue by WillQizza** [CMPUT301W23T09/QRHunter#21]
I feel this needs more clarification, what other QRs are we comparing this to? What does it mean estimate? 
Does it involve us needing some sort of algorithm to optimally get the best QR points??? hek

Eg. 'How many points away from the next rank?'

**Comment by jmmabanta** [comment]
I tried this out on master and at first glance the percentage calculation seems off, but I haven't looked too deep into it

**Comment by jmmabanta** [comment]
Yeah shouldn't getTopQRPosition() have some sort of way to sort by QR code so the position is based on that?

**Comment by andy-mtng** [comment]
This issue should be cleared up based on some discussion John and I had during the lab. For anyone else reading, the calculation was confusing to interpret for everyone, myself included because we were working with very few samples of data.

**Comment by jmmabanta** [comment]
Agreed, it was just a bit off putting due to the very few samples.
Also, should be rounded to nearest 5% to make the number 'easier' to digest (compared to 'Top 62.4213%')

**Comment by andy-mtng** [comment]
Yup I can definitely add the rounding as well along with the data for the other 2 categories (total score, number of codes scanned).
